softwarereview_editor.Rmd

```diff
@@ -117,6 +117,16 @@ Each submission should be reviewed by *two* package reviewers. Although it is fi
   package, contact [an rOpenSci staff member](https://ropensci.org/about/#team) so they might contact the authors
   about transfer to [the `ropensci-books` GitHub organisation](https://github.com/orgs/ropensci-books).
 
+#### Packages remaining in original GitHub organizations
```

Comment:
For PT translation https://github.com/ropensci/dev_guide/pull/855/files#r2018730631

Reply:
Hi @maelle ! I checked and it's on main:

 https://github.com/ropensci/dev_guide/blob/4260c92c9e23d8eb764e0dc59ae8990f3dc82f51/softwarereview_editor.pt.Rmd#L114-L120

Reply:
Awesome, thank you!